### PR TITLE
Docfix: Remove Header Interceptor has malformed table in user guide

### PR DESCRIPTION
**Pull request by simonati**

### CODE REVIEW

File: flume-ng-doc/sphinx/FlumeUserGuide.rst [hunk 1]
```diff
@@ -3920,7 +3920,7 @@ Property Name          Default      Description
 **type**               --           The component type name has to be ``remove_header``
 withName               --           Name of the header to remove
 fromList               --           List of headers to remove, separated with the separator specified with from.list.separator
-fromListSeparator    \s*,\s*      Regular expression used to separate multiple header names in the list specified using from.list
+fromListSeparator      \s*,\s*      Regular expression used to separate multiple header names in the list specified using from.list
```
Comment (adenes) [inline flume-ng-doc/sphinx/FlumeUserGuide.rst]: `\\s*,\\s*` should be in the default column, `\s*,\s*` will be translated to `s*,s*`. (Double backslash is needed to display a single one)

**Review by adenes — CHANGES_REQUESTED** [review]
Thanks for catching and fixing this, I had one minor comment otherwise LGTM

**Comment by bessbd** [comment]
@satiatcloudera : thank you for the patch!

It looks all good to me. Please, fix the issue @adenes has mentioned and I'll commit this.

**Review by bessbd — APPROVED** [review]
LGTM, +1

**Comment by bessbd** [comment]
I'm about to commit this

**Comment by bessbd** [comment]
@simonati : thank you for the patch!

**Comment by bessbd** [comment]
@adenes : thank you for the review!

**Pull request closed without merging**